packetbeat/protos/cassandra/internal/gocql/frame.go

```diff
@@ -443,7 +443,14 @@ func (f *Framer) parseResultPrepared() map[string]interface{} {
 
 	result := make(map[string]interface{})
 
-	result["prepared_id"] = string((f.decoder).ReadShortBytes())
+	uuid, err := UUIDFromBytes((f.decoder).ReadShortBytes())
```

Comment:
UUID is 16 bytes long. Here we are reading the short bytes. Internally ReadShortBytes() will have A [short] n, followed by n bytes if n >= 0. In this case, n will be 16 bytes.

You have mentioned, prepare_id is only two bytes. Can you please add some more details on that?